modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -943,37 +940,14 @@ definition {
 				shippingMethod = "Standard");
 		}
 
-		task ("When the user changes the order status and the action puts order on hold") {
+		task ("Then the action is triggered and the order status is updated") {
 			CommerceNavigator.gotoPortlet(
 				category = "Order Management",
 				portlet = "Orders");
 
-			CommerceNavigator.gotoOrderEntryViaAdmin(status = "Pending");
-
-			Click(
-				key_text = "Accept Order",
-				locator1 = "Link#ANY");
-
-			CommerceOrders.openOrdersAdmin();
-
 			AssertElementPresent(
 				key_value = "On Hold",
 				locator1 = "CommerceEntry#BODY_VERTICAL_ELLIPSIS");
-		}
-
-		task ("Then the action is triggered") {
-			ObjectAdmin.openObjectAdmin();
-
-			ObjectPortlet.selectCustomObject(label = "Commerce Order");
-
-			ObjectAdmin.goToActionsTab();
-
-			ObjectAdmin.assertObjectActions(
-				actionActive = "Yes",
-				actionLabel = "Custom Action",
-				status = "Success");
-
-			// Workaround for LPS-193383
 
 			CommerceOrders.openOrdersAdmin();
```

Comment:
@evanilson-santana These are actions so it would be nice to have them under a `When`, or in this case `and When`

Reply:
The assertions can be under the last `and Then`